tools/src/Tools_Docgen.resi

```diff
@@ -11,31 +11,52 @@ type constructor = {
   signature: string,
   deprecated?: string,
 }
-type detail = Record({items: array<field>}) | Variant({items: array<constructor>})
+@tag("kind")
```

Comment:
We need to add the attributes in the interface files otherwise the compiled one will use the `TAG` field